wordpress-mobile/WordPress-Android
# Latest post summary improvements

Fixes #9068 

This PR separates the `Views` item on top of the chart and moves the `Likes` and the `Comments` items as list items below the chart. 

To test:
* Open Stats/Insights
* Check that the Latest post summary works as it did before

![screenshot_1548428920](https://user-images.githubusercontent.com/1079756/51754056-a58d8800-20bb-11e9-8d42-3719da7469a5.png)
 
Update release notes:
